apollo-router/tests/integration/redis.rs

```diff
@@ -959,26 +978,6 @@ async fn query_planner_redis_update_planner_mode() {
     .await;
 }
 
-#[tokio::test(flavor = "multi_thread")]
-async fn query_planner_redis_update_introspection() {
```

Comment:
This test is removed because it specifically tests that enabling introspection in config changes Redis cache keys. This PR changes `caching_query_planner.rs` for this to no longer be the case, since introspection responses (or "introspection is disabled" errors) are now never in this cache.